src/stores/Tabs/tab_selected_items.js

```diff
@@ -43,6 +44,14 @@ export const TabSelectedItems = types
       }
     },
 
+    get listObject() {
```

Comment:
what's the reason for this getter/setter? can it be just `volatile` or `frozen`, so you can track it?

Reply:
mostly because I havent used them before but digging into this it looks like i should be able to do something like `listObject: types.optional(types.array(types.late(() => DynamicModel.get("TaskModel"))), []),` and in that case it would be better - but im running into some issues - hopefully i'll be able to figure this out shortly